kernel/arch/dreamcast/kernel/timer.c

```diff
@@ -12,6 +12,9 @@
 #include <arch/timer.h>
 #include <arch/irq.h>
 
+/* Bit mask from position */
+#define BIT(n)      (1 << (n))
```

Comment:
OMG I was literally thinking the same thing, but I wasted a bunch of time trying to figure out where the hell to put it and decided to punt on that until later when I changed other code to use this... But hey, now that you're here, what do you think? Where should it live?

Reply:
My brain wasn't quite firing on all cylinders, so I could tell it was wrong but not why. What it should really be is just that the bit shifting is encoded in the field defines. `UNF` is *never* accessed for the purpose of using the number 8, but *always* as the 8th bit so why aren't the field defines just the bitmasks rather than the number of the bit?